### ISSUE TITLE
Enforcing attributes (existence or value)

### ISSUE THREAD

**Issue by codepunkt** [yeonjuan/html-eslint#126]
@yeonjuan Thanks for your great work on this plugin!

There is the `no-restricted-attrs` rule that disallows specified attributes. It would be a great addition to this plugin if it had another, similar rule, `enforce-required-attrs` that would enforce certain attributes (and optionally attribute values) on specific tags.

The use case I'm looking at is enforcing a specific `viewBox` attribute for "svg" nodes for usage in our Icon library, but I'm sure there would be many more use cases where someone might want to enforce an attribute.

What do you think?

**Comment by yeonjuan** [comment]
Hi @codepunkt Thanks for the suggestion! I agree to add a new rule. (maybe `require-attrs`)
I think it would be good to make the rule configuration as follows.

ex)
```js
module.exports = {
   rules: {
       "@html-eslint/require-attrs": ["error",
       {
           tag: "svg",
           attr: "viewBox",
           message: "Missing `viewBox` in svg"
       },
       {
           tag: "img",
           attr: "alt",
           message: "Missing `alt` in img"
       },
        ]
   }
}

```

```html
<svg></svg> // Lint Error: Missing `viewBox` in svg

<img> // Lint Error: "Missing `alt` in img"

```

**Comment by codepunkt** [comment]
@yeonjuan That would be great - you are amazingly fast to respond! 🚀

After taking a look at the PR you opened there's one little thing missing for me personally - which is the option to also compare with a string value.

Like this:
```js
module.exports = {
   rules: {
       "@html-eslint/require-attrs": ["error", {
           tag: "svg",
           attr: "viewBox",
           value: "0 0 32 32",
           message: "`viewBox` in svg tag has wrong value"
       }]
   }
}
```

```html
<svg viewBox="0 0 16 16"></svg> // Lint Error: `viewBox` in svg tag has wrong value
```

**Comment by yeonjuan** [comment]
@codepunkt Thanks for the details! The rule config you suggests is better than mine. I'll change it.
Maybe I can implement autofix feature.

```html
<svg></svg> 

<svg viewBox="0 0 32 32"></svg> // fixed
```
